docker-compose.override.yml

```diff
@@ -0,0 +1,59 @@
+version: '3.3'
+
+services:
+
+  api:
+    build:
+      args:
+        - PYTHON_ENV=dev
+    environment:
+      VIRTUAL_HOST: api.lvh.me
+      VIRTUAL_PORT: 8000
+      SECRET_KEY: secret_key
+      DEBUG: true
```

Comment:
This has to be in quotes to get passed as an env variable to Python b/c otherwise yaml treats it as a boolean:
`true` => `'True'`
C.f. https://github.com/docker/compose/issues/1788